app/main/api/handlers/encryptWallet.ts

```diff
@@ -113,6 +116,31 @@ async function replaceWallet(
   }
 }
 
+/**
+ * Store plain info
+ * @param wallet
+ * @param system
+ */
+async function storePlainInfo(wallet: Wallet, system: AppStorageS & AppStateS) {
+  const walletInfos = asType(
+    system.appStateManager.get("wallets"),
+    Wallets,
+    encryptWalletErrors.UNAVAILABLE_WALLET_INFOS
+  )
+  const infos = {
```

Comment:
I find the name of this variable somewhat confusing (as it contains a field with the same name, infos). I would change it to something more descriptive such as `updatedWalletInfos`